acquire/acquire.py

```diff
@@ -634,16 +634,28 @@ class Recents(Module):
 
 @register_module("--recyclebin")
 class RecycleBin(Module):
-    DESC = "recycle bin metadata"
+    DESC = "recycle bin metadata and data files"
 
     @classmethod
     def _run(cls, target, collector):
         for fs, name, mountpoints in iter_ntfs_filesystems(target):
-            log.info("Acquiring recycle bin metadata from %s (%s)", fs, mountpoints)
+            log.info("Acquiring recycle bin metadata and data files from %s (%s)", fs, mountpoints)
+
+            patterns = [
+                "$Recycle.bin/**/$I*",
+                "Recycler/*/INFO2",
+                "Recycled/INFO2",
+                "$Recycle.Bin/$R*",
+                "$Recycle.Bin/*/$R",
+                "RECYCLE*/D*",
+            ]
 
-            patterns = ["$Recycle.bin/**/$I*", "Recycler/*/INFO2", "Recycled/INFO2"]
             for pattern in patterns:
                 for entry in fs.path().glob(pattern):
+                    if entry.stat().st_size >= (10 * 1024 * 1024):  # 10MB
```

Comment:
You could make this configurable with `@module_arg`.

Reply:
The argument `cli_args` is not passed to `_run` because of that I have adjusted it to `run`, but that is not supported by the `collector`. Does one of functions need an adjustment?

```
Error while running module RecycleBin
Traceback (most recent call last):
  File "/acquire/.tox/py3/lib/python3.10/site-packages/acquire/acquire.py", line 1907, in acquire_target_regular
    mod.run(target, args, collector)
  File "/acquire/.tox/py3/lib/python3.10/site-packages/acquire/acquire.py", line 668, in run
    collector.collect_file(entry, outpath=fsutil.join(name, str(entry)))
  File "/acquire/.tox/py3/lib/python3.10/site-packages/acquire/collector.py", line 289, in collect_file
    raise ValueError("Module name must be provided or Collector needs to be bound to a module")
ValueError: Module name must be provided or Collector needs to be bound to a module
```

Reply:
We should probably change that anyway and pass `cli_args` to `_run`. I don't think there's any reason not to. In fact, I've just created a PR for that: https://github.com/fox-it/acquire/pull/88

Reply:
@Zawadidone if you rebase you now have access to the CLI arguments.